inscriptions/ISic003518.xml

```diff
@@ -192,96 +197,30 @@
-                    valeas.
-                </ab>
-                <!--This EDR text 100007 originally came via the Europeana EAGLE project, http://www.eagle-network.eu/-->
-            </div>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI313635">
-                <ab>
-                    <lb n="1"/>
-
-
-                     <supplied reason="lost">Dis Ma</supplied>nibus
-                    <lb n="2"/>
-                    <supplied reason="lost">— — — — —</supplied>․a
-                    <expan>
-                        <abbr>Ti</abbr>
-                        <ex>beri</ex>
-                    </expan>
-                    <expan>
-                        <abbr>f</abbr>
-                        <ex>ilia</ex>
-                    </expan>
-                    <expan>
-                        <abbr>v</abbr>
-                        <ex>ixit</ex>
-                    </expan>
-                    <expan>
-                        <abbr>a</abbr>
-                        <ex>nnis</ex>
-                    </expan>
-                    XIV
-                    <expan>
-                        <abbr>m</abbr>
-                        <ex>ensibus</ex>
-                    </expan>
-                    V
-                    <lb n="3"/>
-                    <supplied reason="lost">— — — — —</supplied>b<unclear>i</unclear>ll
-                    <unclear>i</unclear>
-                    <supplied reason="lost">․</supplied>m<supplied reason="lost">(?)</supplied>r
-                    <lb n="4"/>
-                    <supplied reason="lost">— — poste</supplied>risque ∙ suis
-                    <lb n="5"/>
-                    <supplied reason="lost">— — — — —</supplied>․ moneta ∙
-                    <lb n="6"/>
-                    <supplied reason="lost">— — — — —</supplied>
-                    valeas.
-                    <lb n="7"/>
+                    <lb n="1"/><w><supplied reason="lost">Dis</supplied></w> <w><supplied reason="lost">Ma</supplied>nibus</w>
+                    <lb n="2"/><gap reason="lost" extent="unknown" unit="character"/><persName type="attested"><name type="nomen"><seg part="F">a</seg></name> <name type="patronym"><expan><abbr>Ti</abbr><ex>berii</ex></expan></name> <w><expan><abbr>f</abbr><ex>ilia</ex></expan></w></persName> <g ref="#interpunct"/> <w><expan><abbr>v</abbr><ex>ixit</ex></expan></w> <g ref="#interpunct"/> <w><expan><abbr>a</abbr><ex>nnis</ex></expan></w> <g ref="#interpunct"/> <num value="14">XIV</num> <g ref="#interpunct"/> <w><expan><abbr>m</abbr><ex>ensibus</ex></expan></w> <g ref="#interpunct"/> <num value="5">V</num>
+                    <lb n="3"/><gap reason="lost" extent="unknown" unit="character"/><unclear>b</unclear><unclear>i</unclear><unclear>l</unclear><unclear>l</unclear><unclear>i</unclear> <gap reason="lost" extent="unknown" unit="character"/>M <gap reason="lost" extent="unknown" unit="character"/>R
```

Comment:
Those single letters 'M' and 'R' should go in `orig` as in the previous example.